.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ exclude: |
     python/ray/tests/test_cli_patterns|
     python/ray/_private/runtime_env/_clonevirtualenv.py|
     release/release_logs/|
+    thirdparty/patches/
```

Comment:
exclude this folder as mentioned in https://github.com/ray-project/ray/pull/50188#discussion_r1941556952